app/templates/admin/bonnerManagement.html

```diff
@@ -15,7 +15,7 @@
 {% block app_content %}
 
     {# Figure out if we are showing a particular tab inside of the accordion #}
-    {% set subTab =  visibleAccordion[7:] if visibleAccordion and "cohort-" in visibleAccordion else "newest" %}
+    {% set subTab =  visibleAccordion[7:] if visibleAccordion and "cohort-" in visibleAccordion and visibleAccordion[7:]|int <= cohorts.keys()|sort|last else "newest" %}
```

Comment:
Got a bit of a weird space after the = here.

Reply:
good catch!